index.js

```diff
@@ -2,18 +2,4 @@
 
 var path = require('path');
 
-var escape = function (str) {
-    return str.replace(/[\[\]\/{}()*+?.\\^$|-]/g, '\\$&');
-};
-
-var regexp = ['src', 'test'].map(function (i) {
-    return '^' + escape(path.join(__dirname, i) + path.sep);
-}).join('|');
-
-require('babel/register')({
-    only:   new RegExp('(' + regexp + ')'),
-    ignore: false,
-    loose: 'all'
-});
-
 module.exports = require('./src');
```

Comment:
Is it necessary to fix path to the dir here? Looks like we will get `dist` after compile?